docs/v3.x/plugins/upload.md

```diff
@@ -107,6 +107,9 @@ To upload files that will be linked to a specific entry.
 - `source` (optional): The name of the plugin where the model is located.
```

Comment:
The related information you are making a note is located just above the note.

Thank you for the PR but I don't think this is needed.

Reply:
I don't think this is well enough explained. From a user perspective I think it would be nice to know that source isn't actually optional but mandatory otherwise it'll break the api. Anyway if you guys think this is clear enough, go for it! :)

Reply:
I think the problem is more so by not knowing that the _user_ collection-type is part of the user-permissions plugin (default installed). If you guys knew it was part of a plugin, you would understand that you needed the `source` param.

Reply:
I agree with your argument, however I still think it won't harm if we notify the user. The User model is requiring the source since it depends on the user-permissions plugin. Right now it's a bit vague when it's required and when it's not.

Besides if the source is not used, some generic errors arose which were non explanatory to me. 

Do you agree?